core/pom.xml

```diff
@@ -2,7 +2,7 @@
   <modelVersion>4.0.0</modelVersion>
   <parent>
     <groupId>io.parsingdata</groupId>
-    <artifactId>metal-parent</artifactId>
+    <artifactId>metal</artifactId>
```

Comment:
Sneaky :wink: I think we won't have any troubles releasing this next version, but it's something to keep in mind if we do.